### PR TITLE
Add documentation for addTask, mark and unmark in user guide

### PR DESCRIPTION
**Pull request by shayaansultan**
Update user guide with format and examples of addTask, mark and unmark tasks functionality

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -90,6 +90,56 @@ Examples:
...
+- The `uid` refers to the user ID displayed beside the employee's name.
+- The `taskIndex` refers to the index number shown in the displayed task list.
+- The `taskIndex` **must be a positive integer** 1, 2, 3, …​
+
+Examples:
+
+- `unmark uid/1 2` unmarks the 2nd task in the task list of the employee with the `uid` of 1 as not completed.
+- `unmark uid/2 1` unmarks the 1st task in the task list of the employee with the `uid` of 2 as not completed.
+
+
```
Comment (PyromancerBoom) [inline docs/UserGuide.md:142]: Well explained, easy to grasp, and quite well-formatted UG as well! The examples are quite helpful

**Review by PyromancerBoom — COMMENTED** [review]
LGTM!

File: docs/UserGuide.md (hunk 1 above)
Comment (quelinxiao) [inline docs/UserGuide.md:142]: Great use of examples!

**Review by quelinxiao — COMMENTED** [review]
LGTM

**Pull request merged**